draconic/interpreter.py

```diff
@@ -610,6 +610,17 @@ def _starred_unwrap(self, nodes):
                     raise TypeError(f"Value after * must be iterable, not {type(evalue)}")
             else:
                 yield self._eval(node)
+                
+    def _star_starred_unwrap(self, zipped):
+        for entry in zipped:
+            evalue = self._eval(entry[1])
+            if isinstance(evalue, dict):
+                iterable = iter(evalue.items())
+                sentinel = object()
+                while (retval := next(iterable, sentinel)) is not sentinel:
+                    yield retval[0], retval[1]
+            else:
+                yield entry[0], evalue
```

Comment:
```suggestion
        for key, value in zipped:
            evalue = self._eval(value)
            if isinstance(evalue, dict):
                iterable = iter(evalue.items())
                sentinel = object()
                while (retval := next(iterable, sentinel)) is not sentinel:
                    yield retval[0], retval[1]
            else:
                yield self._eval(key), evalue
```

The returned key needs to be evaluated if it isn't part of the 'starred' dict.
Also, splitting up `entry[0]` and `entry[1]` can be done with the